builder/amazon/chroot/communicator.go

```diff
@@ -114,6 +114,10 @@ func (c *Communicator) UploadDir(dst string, src string, exclude []string) error
 	return err
 }
 
+func (c *Communicator) DownloadDir(src string, dst string, exclude []string) error {
+    return fmt.Errorf("panic")
```

Comment:
Can we change these to either:

    panic("not implemented")

Or

    return fmt.Errorf("DownloadDir is not implemented for amazon-chroot")

The panic shows a file and line and this error is easy to grep for so if we want to go back and add the implementation it's easy to find the right spot.

I think either panic or error is OK but let's use the same style everywhere.